### PR TITLE
V9_3-HTCONDOR-616-branch

### PR DESCRIPTION
**Pull request by Todd-L-Miller**
Implement the common cloud attributes.

### CODE REVIEW

File: src/condor_utils/param_info.in [hunk 1]
```diff
@@ -6702,5 +6702,32 @@ default : @end
...
+[$FEATURE.JobsHaveInstanceIDs]
+description=When a job runs on a machine in the cloud, add that machine's instance ID to the job's ClassAd
+default : @end
+	SYSTEM_JOB_MACHINE_ATTRS = $(SYSTEM_JOB_MACHINE_ATTRS) CloudInstanceID
+@end
+
+[$FEATURE.CommonCloudAttributesAWS]
+description=Enable an execute node to advertise common cloud attributes on AWS
+default : @end
+    if $(1?)
```
Comment (johnkn) [inline src/condor_utils/param_info.in:6725]: if statement not necessary

File: src/condor_utils/param_info.in [hunk 2]
```diff
@@ -6702,5 +6702,32 @@ default : @end
 	DAEMON_LIST=$(DAEMON_LIST),ADSTASH
 @end
 
+[$FEATURE.CommonCloudAttributesGoogle]
+description=Enable an execute node to advertise common cloud attributes on Google
+default : @end
+	if $(1?)
```
[the remark this answers is not in the document]
Comment (Todd-L-Miller) [reply src/condor_utils/param_info.in:6708]: TJ confirms that `use FEATURE : StartdCronOneShot(Cloud, exe, )` and `use FEATURE : StartdCronOneShot(Cloud, exe)` behave the same.

TJ further points out that there's no reason to limit ourselves to just one optional argument there, and we might as well be passing `$(0)`.

File: src/condor_utils/param_info.in (hunk 1 above)
Comment (Todd-L-Miller) [reply src/condor_utils/param_info.in:6725]: See previous comment.

**Review by johnkn — APPROVED** [review]
looks good to me.

**Pull request merged**